scripts/docker/build-and-publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+REGION=us-east-1
+AWS_ACCOUNT_ID=443280315061
```

Comment:
Accaunt id is secure information and can be used to hack your account. Please try to use env variables and not to commit it in VCS